### ISSUE TITLE
Possible solution for the maximum number of requests limit

### ISSUE THREAD

**Issue by EusebiuOprinoiu** [afragen/github-updater#168]
Hello, Andy!
I haven't studied your code yet to see how often you check for new versions on GitHub, but if this is something that is already implemented, ignore the thread.
Since GitHub has a limit of 60 API requests per hour I think there should be some preventive measures in place. For example, you could check if new updates are available only once or twice a day. This way, the API limit will not be reached if we have multiple sites/test environments on the same server.
In addition to that you could add a button on the plugin page that allows to make a manual check. ( Very useful on development when we make changes often )

Best regards,
Eusebiu Oprinoiu

**Comment by afragen** [comment]
Hey Eusebiu,

I make heavy use of the Transient API. All data is cached for 12 hours. If you click the **Check Again** button once main update page all transients will be deleted. Take a glance at the README for more goodies. :wink:

**Comment by EusebiuOprinoiu** [comment]
Thank you, Andy! I don't know how I missed that when I read the documentation.. :)

**Comment by EusebiuOprinoiu** [comment]
I don't want to bug you with another issue, so I'm going to post here since it's somehow related.
Pressing the Check Again button doesn't seem to work when JetPack is enabled.
I also tried removing all transients using Wp Optimize and the available update still doesn't show up.

**Comment by afragen** [comment]
JetPack doesn't interfere with updates displaying but I have found that sometimes I have to reload the update page a couple of times. All the data for updates is present but for some reason it doesn't load every time.
